packages/react-native/src/components/EvervaultProvider.tsx

```diff
@@ -36,30 +35,34 @@ interface EvervaultProps {
  * }
  * ```
  */
-const EvervaultProvider = ({ teamUuid, appUuid, children }: EvervaultProps) => {
+const EvervaultProvider = ({ teamId, appId, children }: EvervaultProps) => {
   useEffect(() => {
     async function initEvervault() {
       try {
-        if (!teamUuid?.startsWith("team_")) {
+        if (!teamId?.startsWith("team_")) {
```

Comment:
Actually just thinking, some of the older teams don't have this prefix so they wouldn't be able to use this

Reply:
good point will remove :(